scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -1314,6 +1362,8 @@ def extra_config_as_object(self, force_reload: bool = False) -> EnvironmentConfi
                     self._extra_config = EnvironmentConfig(extra_config_with_path)
                 elif self.type == AssetType.MODEL:
                     self._extra_config = ModelConfig(extra_config_with_path)
+                elif self.type == AssetType.DATA:
+                    self._extra_config = DataConfig(extra_config_with_path)

```

Comment:
Some (small) data assets only use local storage. How should they be handled? Here's one example: https://github.com/Azure/azureml-assets/blob/6446f5fd809a57a4e39cab1d7c56e2c9a809efd6/assets/promptflow/data/ask-wikipedia/asset.yaml

Reply:
discussed via chat, no extra config will be created for local storage